.github/workflows/test-tgc.yml

```diff
@@ -0,0 +1,98 @@
+name: TGC Build and Unit Test
```

Comment:
TGC is the only slug I could think of. 

Terraform Google Conversion is too long and out strips the suffix.